### ISSUE TITLE
[BUG] Only one highlight per page is added

### ISSUE THREAD

**Issue by mswitakowski** [hadynz/obsidian-kindle-plugin#145]
I found that the plugin adds only one highlight if there are multiple highlights on the same page. Consider the following input:

```
Oreilly.Developing.Backbone.js.Applications.Apr.2012 (Addy Osmani)
- Highlight on Page 7 | Added on Monday, 3 December 12 19:42:30 Greenwich Mean Time

JavaScript templating libraries (such as Handlebars.js or Mustache)
==========
Oreilly.Developing.Backbone.js.Applications.Apr.2012 (Addy Osmani)
- Highlight on Page 10 | Added on Monday, 3 December 12 19:50:09 Greenwich Mean Time

The GoF (Gang of Four)
==========
Oreilly.Developing.Backbone.js.Applications.Apr.2012 (Addy Osmani)
- Highlight on Page 10 | Added on Monday, 3 December 12 19:50:51 Greenwich Mean Time

a variation of three other classical design patterns: the Observer (Pub/Sub), Strategy and Composite patterns
==========
Oreilly.Developing.Backbone.js.Applications.Apr.2012 (Addy Osmani)
- Highlight on Page 10 | Added on Monday, 3 December 12 19:51:33 Greenwich Mean Time

Factory and Decorator patterns
==========
```

The plugin will add one highlight on page 7 and only one on page 10. Manually editing the input to contain unique page numbers addresses the issue.

## Highlights
JavaScript templating libraries (such as Handlebars.js or Mustache) — location: [7]() ^ref-5204

---
Factory and Decorator patterns — location: [10]() ^ref-40138

**Desktop (please complete the following information):**
 - OS: [Linux]
 - Plugin version [1.5.2]
 - Obsidian version [0.13.23]

**Comment by marc0l92** [comment]
This pull request should fix this issue:
https://github.com/hadynz/KindleClippings/pull/10

**Comment by hadynz** [comment]
Thanks for raising this rather important bug!

Fixed in [1.5.3][1]. If you can test and confirm that it is working for you, that would be great. 

[1]: https://github.com/hadynz/obsidian-kindle-plugin/releases/tag/1.5.3
